src/test/java/uk/gov/hmcts/reform/sscs/ccd/service/CcdCallbackMapServiceTest.java

```diff
@@ -105,24 +126,49 @@ void handleCcdCallbackPostCallbackDwpState() {
     @Test
     void handleCcdCallbackV2PostCallbackDwpState() {
         given(callbackMap.getPostCallbackDwpState()).willReturn(DIRECTION_ACTION_REQUIRED);
-        given(callbackMap.getCallbackEvent()).willReturn(READY_TO_LIST);
+        given(callbackMap.getCallbackEvent()).willReturn(EventType.READY_TO_LIST);
+        given(callbackMap.getCallbackSummary()).willReturn("summary");
+        given(callbackMap.getCallbackDescription()).willReturn("description");
+        given(idamService.getIdamTokens()).willReturn(idamTokens);
+        given(updateCcdCaseService
+                .updateCaseV2(eq(CASE_ID), eq(EventType.READY_TO_LIST.getCcdType()), eq("summary"), eq("description"), eq(idamTokens), any(Consumer.class)))
```

Comment:
Might be worth cleaning up the static imports for event type in this class
Also any blank spaces/tabs introduced if thats appropriate